src/app/helptext/vm/vm-wizard/vm-wizard.ts

```diff
@@ -64,8 +64,7 @@ export default {
  address can be chosen.'),
 
   vcpus_label: T('CPU and Memory'),
-  vcpus_warning: T('The product of vCPUs, cores and threads must not exceed'),
-  vcpus_warning_b: T('on this system.'),
+  vcpus_warning: T('The product of vCPUs, cores and threads must not exceed {maxVCPUs} on this system.'),
```

Comment:
IMHO, we can use the same way for globalHelptext as you did here. If I am wrong, please let me know.

Reply:
The `globalHelptext.Ctrlr` is a static string, where `maxVCPUs` is a variable. We could do this way but it doesn't make sense, since it'd require additional code in components to pass the same string.
Something like:
```diff
- activedirectory_netbiosname_b_placeholder: T(`Netbios Name (${globalHelptext.Ctrlr} 2)`)
+ activedirectory_netbiosname_b_placeholder: T('Netbios Name ({controller} 2')
...
- placeholder: helptext.activedirectory_netbiosname_b_placeholder
+ placeholder: this.translate.instant(helptext.activedirectory_netbiosname_b_placeholder, { controller: globalHelptext.Ctrlr });
...